src/padding.jl

```diff
@@ -277,8 +277,127 @@ function pad_reflect(x::AbstractArray{F,N}, pad::NTuple{2,Int};
   return cat(xl, x, xr, dims = dims)
 end
 
+"""
+    pad_symmetric(x, pad::Tuple; [dims])
+    pad_symmetric(x, pad::Int; [dims])
+
+Pad the array `x` reflecting its values symmetrically across the border, i.e. the border values of `x` are present in the padding values, in contrast to [`pad_reflect`](@ref).
+
+`pad` can a tuple of integers `(l1, r1, ..., ln, rn)`
+of some length `2n` that specifies the left and right padding size
+for each of the dimensions in `dims`. If `dims` is not given, 
+it defaults to the first `n` dimensions.
+
+For integer `pad` input instead, it is applied on both sides
```

Comment:
I don't quite parse what "instead" means in this sentence. Maybe leftover word from an earlier edit?

Reply:
Oh, it was copied from another docstring. I think "if `pad` is an integer" would be clearer, but the boilerplate we have around dims handling could use some wordsmithing in general. @CarloLucibello WDYT?

Reply:
Yeah, it was copied from the pad_repeat docstring. I agree "if pad is an integer" would be clearer.

Reply:
Feel free to change as you see fit if it makes things clearer :)